PracticeTask1/Worker.cs

```
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PracticeTask1
+{
+    public class Worker
+    {
+        public string Name { get; set; }
+        public int Salary { get; set; }
+
+        private int age;
+
+        public int Age
+        {
+            get
+            {
+                return age;
+            }
+            set
+            {
+                if ((value > 0) && (value < 101))
```

Lets throw an exception if value is invalid.